src/odemis/gui/win/acquisition.py

```diff
@@ -1008,6 +1061,21 @@ def on_acquisition_done(self, future):
         self.terminate_listeners()
         self.EndModal(wx.ID_OPEN)
 
+    def on_tiles_number(self, _=None):
+        """
+        Called when the user enters values for the tiles number in the GUI.
+        """
+        nx = self.tiles_nx.value 
+        ny = self.tiles_ny.value 
+        w = nx*self.fov[0]-(nx-1)*self.fov[0]*self.overlap
+        h = ny*self.fov[1]-(ny-1)*self.fov[1]*self.overlap
```

Comment:
Did you check `nx * self.fov[0] * (1 - self.overlap)`? That should then be the same formula as tiledacq.

Reply:
This formula alone is not enough. As you said, the number of tiles is some number followed by `.00004` for example, then its taking the ceil value increases the number of tiles. So the suggestion of decreasing the size area seems effective